<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewUI</class>
 <widget class="QWidget" name="PreviewUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,25,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wDisplayElements" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,255,255);</string>
     </property>
     <layout class="QHBoxLayout" name="hlDisplayElements">
      <item>
       <widget class="QLabel" name="laZones">
        <property name="text">
         <string>Affichage des zones (uniquement dans l'aperçu) :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbZoneTitles">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox
{
	color : rgb(0,121,0);
	font-style: normal;
	font-size: 8pt;
	font-weight: bold;
}

</string>
        </property>
        <property name="text">
         <string>titres</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbZoneExternMargins">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
QCheckBox
{
	color : rgb(151,0,0);
	font-style: normal;
	font-size: 8pt;
	font-weight: bold;
}

</string>
        </property>
        <property name="text">
         <string>marges extérieures</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbZoneConsignes">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
QCheckBox
{
	color : rgb(63,72,204);
	font-style: normal;
	font-size: 8pt;
	font-weight: bold;
}

</string>
        </property>
        <property name="text">
         <string>textes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbZoneInternMargins">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
QCheckBox
{
	color : rgb(221,88,0);
	font-style: normal;
	font-size: 8pt;
	font-weight: bold;
}

</string>
        </property>
        <property name="text">
         <string>marges intérieures</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbZonePhotos">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
QCheckBox
{
	color : rgb(112,124,190);
	font-style: normal;
	font-size: 8pt;
	font-weight: bold;
}

</string>
        </property>
        <property name="text">
         <string>images</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hsZones">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlPreviewImage">
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlActions" stretch="0,1,1">
     <item>
      <spacer name="hsActions">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbGeneration">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
       </property>
       <property name="text">
        <string>Sauvegarde du PDF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbOpenPDF">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
       </property>
       <property name="text">
        <string>Ouvrir dernier PDF</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
